community/newsletter/2022-6.md

```diff
@@ -41,7 +41,7 @@ permalink: /community/newsletter/2022-6/
 
 						<h2 style="font-family: 'Roboto', Helvetica,sans-serif; font-size: 26px; color: rgba(0,0,0,0.87); line-height: 21px; font-weight: 400; text-align: left; margin-bottom: 0px; margin-top: 40px;;">Latest release: Ballerina 2201.1.1 (Swan Lake Update 1)</h2>
 
-						<p style="font-family: 'Roboto', Helvetica,sans-serif;font-size: 14px;color: rgba(0,0,0,0.87);line-height: 21px;font-weight: 400;text-align: left;margin-top: 10px;"><a class="linkbody" href="https://ballerina.io/downloads/?utm_source=mailer&amp;utm_medium=email&amp;utm_campaign=mailer_email_ballerina_nwsltr_july22_220728" style="color: #20b6af;text-decoration: underline;" target="_blank">Swan Lake 2201.1.1</a> is the first patch release of Ballerina 2201.1.0 (Swan Lake Update 1) and includes a new set of bug fixes to the compiler, runtime, standard library, and developer tooling.</p>
+						<p style="font-family: 'Roboto', Helvetica,sans-serif;font-size: 14px;color: rgba(0,0,0,0.87);line-height: 21px;font-weight: 400;text-align: left;margin-top: 10px;"><a class="linkbody" href="https://ballerina.io/downloads/?utm_source=mailer&amp;utm_medium=email&amp;utm_campaign=mailer_email_ballerina_nwsltr_july22_220728" style="color: #20b6af;text-decoration: underline;" target="_blank">Swan Lake 2201.1.1</a> is the first patch release of Ballerina 2201.1.0 (Swan Lake Update 1) and includes a new set of bug fixes to the compiler, runtime, Ballerina library, and developer tooling.</p>
 
```

Comment:
Do we need to change the old ones?

Reply:
I think this question is relevant to many files I guess.

Reply:
@keizer619 @anupama-pathirage

Reply:
Yes, Agreed with @shafreenAnfar here. We don't need to change old places like newsletters, published blogs etc. They are already sent over mail/published (ppl have seen those already) This is a new terminology we are using and we need to update all learn pages/home page/api docs or whatever current content only.